src/applications/beta-enrollment/manifest.json

```diff
@@ -1,8 +1,5 @@
 {
   "appName": "Beta Enrollment",
   "entryFile": "./beta-enrollment-entry.jsx",
-  "entryName": "beta-enrollment",
-  "rootUrl": "/beta-enrollment",
-  "production": true,
-  "contentPage": "beta-enrollment/personalization.md"
+  "entryName": "beta-enrollment"
```

Comment:
I think @erikphansen is using the beta enrollment for the direct deposit stuff.

Reply:
nope, we aren't hiding the `profile-beta` page content behind a beta page, but thanks for brining it up!

Reply:
Yeah, this would only affect applications that use `beta-enrollment` as the `entryname`. This should probably just be a Platform component loaded up as a static-page widget via the `static-pages` app entry point. It's weird having it as its own bundle.